app/src/lib/stores/app-store.ts

```diff
@@ -2198,7 +2209,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
     const diff = await getWorkingDirectoryDiff(
       repository,
       selectedFileBeforeLoad,
-      enableHideWhitespaceInDiffOption() && this.hideWhitespaceInDiff
+      enableHideWhitespaceInDiffOption() && this.hideWhitespaceInChangesDiff
```

Comment:
Won't `hideWhitespaceInChangesDiff` be false if feature flag is false? (Is check needed here since we feature flag the entry point in UI?)

Reply:
This is the case, but the reason for the flag check here is to allow us to rollback the flag if needed.

Reply:
Ah ok. Cool